float64 gcvX
float64 gcvY
float64 gcvZ
bool walk
bool kick